### PR TITLE
Update aggregate.js

### PR DESCRIPTION
**Pull request by smohanty92**
Fixing unset issue due to Meteor updating clientCollection to be Map instead of Object

Ref: https://github.com/meteor/meteor/commit/79ae1849f5846e207ed4dfa49831ba409252fd03#diff-173e69ea0353a765b98017d67abc45ec7ce1449178466dc74738324db83f9183

### CODE REVIEW

**Review by robfallows — CHANGES_REQUESTED** [review]
Needs the following:

1. Version bump in `package.js`
1. Description of change in `History.md` (basically the notes in your PR).

**Comment by smohanty92** [comment]
Please see update. Thanks!

**Comment by robfallows** [comment]
As I understand it, this change will break in Meteor prior to v2.6. That will require additional work to ensure that the current version is still available for earlier builds.

Potentially breaking changes require a semver major version change. However, Meteor's dependency management will allow a minor version change (in this case, from `1.3.6` to `1.4.0` as long as the following steps are done:

1. The [`package.js` line](https://github.com/robfallows/tunguska-reactive-aggregate/blob/6718deb2cca11722cd59b92fb6559121961637c0/package.js#L10) responsible for controlling the version update will need changing to `2.6`.
1. Testing will need to be done to confirm that not only does it work as intended in Meteor 2.6, but that older versions of Meteor will not attempt to update this package to `2.0.0` and consequently break.

**Comment by smohantyCME** [comment]
Are we sure this will affect any app running Meteor below V2.6? 
I'm seeing that change from Obect->Map and I'm on version 2.3.

The PR for the update to Meteor was in July, 2018. 

https://github.com/meteor/meteor/pull/10053

I tried following Meteor's changelog to see what version # this actually got introduced in but couldn't find it. However, comments in the PR appear to show it may have been around V1.8.1

File: History.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Change History
 
+## v1.3.7 2022-01-18
+
+- Fixing unset issue due to Meteor updating clientCollection to be Map instead of Object
+- Ref: meteor/meteor@79ae184#diff-173e69ea0353a765b98017d67abc45ec7ce1449178466dc74738324db83f9183
```
Comment (robfallows) [inline History.md:6]:
```suggestion
  - Ref: [DDP Package Changes](meteor/meteor@79ae184#diff-173e69ea0353a765b98017d67abc45ec7ce1449178466dc74738324db83f9183)
```

**Review by robfallows — CHANGES_REQUESTED** [review]
One small suggestion to improve readability.

**Comment by smohanty92** [comment]
All set, sorry for switching of accounts.

**Review by robfallows — APPROVED** [review]
LGTM 👍

**Pull request merged**